source/Halibut.Tests/Transport/DiscoveryClientFixture.cs

```diff
@@ -44,7 +44,7 @@ public async Task DiscoveringNonExistentEndpointThrows(SyncOrAsync syncOrAsync)
 #pragma warning disable CS0612
             await syncOrAsync
                 .WhenSync(() => Assert.Throws<HalibutClientException>(() => client.Discover(fakeEndpoint), "No such host is known")).IgnoreResult()
-                .WhenAsync(async () => await AssertAsync.Throws<HalibutClientException>(() => client.DiscoverAsync(fakeEndpoint, CancellationToken), "No such host is known"));
+                .WhenAsync(async () => await AssertAsync.Throws<HalibutClientException>(() => client.DiscoverAsync(fakeEndpoint, new HalibutTimeoutsAndLimits(), CancellationToken), "No such host is known"));
```

Comment:
Do we want a helper that sets some good defaults instead of `new HalibutTimeoutsAndLimits()` so it's consistent with the statics we set at the moment to keep some consistency amount how timeouts are set. Individual tests can choose to change them, otherwise they all get the same.

Reply:
Ahh, I see they are defaulted from the statics so it is doing that 👍

Reply:
`HalibutTimeoutsAndLimits` get its defaults from what is set statically which will be the reduced values as part of `LowerHalibutLimits`. In theory those should be "good" defaults.